src/js/modules/articles/components/ArticleBody.js

```diff
@@ -117,39 +99,30 @@ const ArticleBody = ({ classes, articles, sections, content, media }) => {
   }
   */
   //  featuredMedia = Object.values(articleMedia).find(image => image.isFeatured);
-  const generateArticleReefer = content => {
-    const match = SPEC_REEFER_PATTERN.exec(content);
-    const article = articles[parseInt(match[1])];
-    console.log(article.title)
-    if (article) {
-      const title = article.title.replace('“', '‘').replace('”', '’');
-      return (
-        <span id="article-reefer">
-          This article was written in response to &ldquo;
-          <Link
-            id="reefer-link"
-            target="_blank"
-            to={`${sections[article.sectionId].permalink}/${article.slug}`}
-          >
-            {title}
-          </Link>
-          ,&rdquo; published in Volume {article.volume} Issue{" "}
-          {article.issue}.
-        </span>
-      );
-    }
-  };
-  const featuredMedia = media.find(medium => medium.isFeatured);
+
+  const isCarouselButtonVisible =
+    SPEC_IMG_CAROUSEL_PATTERN.test(content) && Object.values(media).length > 0;
+  const referencedArticleId = SPEC_REFERENCE_PATTERN.test(content)
+    ? parseInt(SPEC_REFERENCE_PATTERN.exec(content)[1])
+    : -1;
   return (
     <Row>
       <Col xs={12} sm={12} md={8} lg={8} className={classes.ArticleBody}>
-        {featuredMedia && (
+        {isCarouselButtonVisible && (
```

Comment:
I'm not following this boolean's logic. Why does the carousel button's visibility determine the lightbox's visibility? Shouldn't it be the other way around?